client/actions/project.ts

```diff
@@ -137,3 +137,24 @@ export async function getWorkCapacity(projectSlug: string) {
 
     return data;   
 }
+
+export async function getEstimateEffectiveness(milestoneId: string) {
+    const Response = z.array(z.object({
+        storyTitle: z.string(),
+        effectiveness: z.number()
+    }))
+    
+
+    const url = `http://localhost:8080/api/${milestoneId}/getEstimateEffectiveness`;// need to change the url
```

Comment:
please avoid hardcoding. this will now become a bug since we have a deployed site